### PR TITLE
make catatonit `docker-info` compatible

### PR DESCRIPTION
**Pull request by vrothberg**

### CODE REVIEW

File: catatonit.c [hunk 1]
```diff
@@ -242,6 +242,12 @@ static int reap_zombies(void)
 
 int main(int argc, char **argv)
 {
+	/* Print version if --version is the only argument */
+	if (argc == 2 && strcmp("--version", argv[1]) == 0) {
```
Comment (cyphar) [inline catatonit.c]: You need to `#include <string.h>`

```
catatonit.c: In function ‘main’:
catatonit.c:246:19: warning: implicit declaration of function ‘strcmp’ [-Wimplicit-function-declaration]
  if (argc == 2 && strcmp("--version", argv[1]) == 0) {
                   ^~~~~~
```

File: catatonit.c [hunk 2]
```diff
@@ -89,22 +91,22 @@ static void help(void)
...
+	fprintf(stderr, "  -V, --version   Print version information.\n");
 	fprintf(stderr, "\n");
 	fprintf(stderr, "The source code can be found at <%s>.\n", PROGRAM_URL);
 	fprintf(stderr, "For bug reporting instructions, please see: <%s>.\n", PROGRAM_BUGURL);
 }
 
 static void version(void)
 {
-	fprintf(stderr, "%s version %s\n", PROGRAM_NAME, PROGRAM_VERSION);
+	fprintf(stdout, "tini version %s_%s\n", PROGRAM_VERSION, PROGRAM_NAME);
```
Comment (tboerger) [inline catatonit.c]: tini version is intentional?

Comment (vrothberg) [reply catatonit.c]: Yes, that's intentional (unfortunately). Docker hard-codes the parsing of `docker-init --version` to tini. We either do this or we keep an out-of-tree patch for our docker packages, and we preferred this option as we consider it to be more portable.

Comment (cyphar) [reply catatonit.c]: Technology is amazing! :smile_cat:

Comment (tboerger) [reply catatonit.c]: Omg, thanks for the clarification

**Pull request merged**